### PR TITLE
smoke test

### PR DESCRIPTION
**Pull request by rkondratowicz**
test creating a new letter and checking its status

### CODE REVIEW

File: src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/CreateLetterSmokeTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        givenJwt(jwt)
+            .when()
+            .get("/letters/" + id)
+            .then()
+            .statusCode(200);
+    }
+
+    private RequestSpecification givenJwt(String jwt) {
+        return RestAssured
+            .given()
```
Comment (nitinprabhu) [inline src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/CreateLetterSmokeTest.java]: will it work without `relaxedHTTPSValidation()`?

File: src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/SmokeTestSuite.java [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+    @Value("${s2s-secret}")
+    protected String s2sSecret;
+
+    /**
+     * Sign in to s2s.
+     * @return s2s JWT token.
+     */
+    protected String signIn() {
+        return RestAssured
+            .given()
```
Comment (nitinprabhu) [inline src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/SmokeTestSuite.java]: will it work without `relaxedHTTPSValidation()`?

File: src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/CreateLetterSmokeTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.UUID;
+
+public class CreateLetterSmokeTest extends SmokeTestSuite {
+
+    @Test
+    public void should_create_letter_successfully() throws Exception {
+
+        String jwt = signIn();
+
+        String letterJson = new JSONObject()
```
Comment (lgonczar) [inline src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/CreateLetterSmokeTest.java]: I'd extract this bit to a separate method, but up to you.

File: src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/CreateLetterSmokeTest.java (hunk 1 above)
Comment (rkondratowicz) [reply src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/CreateLetterSmokeTest.java]: added

**Pull request merged**